Src/FluentAssertions/Primitives/TimesConstraint.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using FluentAssertions.Common;
+using FluentAssertions.Execution;
+
+namespace FluentAssertions.Primitives
```

Comment:
I agree that this must be part of the `FluentAssertions` namespace and thus needs to be in the root of the project folder.